mode/examples/Basics/Math/Map/Map.pyde

```diff
@@ -0,0 +1,27 @@
+'''
+ Map.
+ 
+ Use the map() function to take any number and scale it to a new number 
+ that is more useful for the project that you are working on. For example, use the
+ numbers from the mouse position to control the size or color of a shape. 
+ In this example, the mouse’s x-coordinate (numbers between 0 and 360) are scaled to
```

Comment:
In this example, the mouse’s x-coordinate (a number between 0 and 360) is scaled to